### PR TITLE
install by recipe

### PR DESCRIPTION
**Pull request by BeachWang**
根据用户的recipe，安装算子的最小依赖。

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,6 +197,18 @@ The dependency options are listed below:
...
+- Install dependencies for specific OPs
+
+With the growth of the number of OPs, the dependencies of all OPs becomes very heavy. Instead of using the command `pip install -v -e .[sci]` to install all dependencies,
+we provide two alternative, lighter options:
+
+  - Automatic Minimal Dependency Installation: During the execution of Data-Juicer, minimal dependencies will be automatically installed. This allows for immediate execution, but may potentially lead to dependency conflicts.
+
+  - Manual Minimal Dependency Installation: To manually install minimal dependencies tailored to a specific execution configuration, run the following command:
+    ```shell
+    python tools/install_by_recipe.py --config path_to_your_data-juicer_config_file
```
Comment (yxdyc) [inline README.md:209]: To keep consistent to Listing 1 of our DJ2.0 paper, rename it into `dj-install`

**Review by yxdyc — COMMENTED** [review]
LTGM, see the minor comment

File: README.md (hunk 1 above)
Comment (HYLcool) [reply README.md:209]: For reference: https://github.com/modelscope/data-juicer/blob/a2612cdff70fcc282f31a62116d8837ebf551a3f/setup.py#L68-L72

**Review by HYLcool — APPROVED** [review]
LGTM

**Pull request merged**